### PR TITLE
S-2384 Change bank calculator

### PR DESCRIPTION
**Pull request by ssinica**

### CODE REVIEW

File: site-cms/wordpress/wp-content/themes/finansportalen/external/js/banking/common/banking-services.js [hunk 1]
```diff
@@ -55,8 +55,6 @@ angular.module('banking')
 				throw "Unable to retrieve result";
 			});
 		}
-	};
```
Comment (ssinica) [inline site-cms/wordpress/wp-content/themes/finansportalen/external/js/banking/common/banking-services.js]: Reviewer, pls, ignore these change. I will not commit it.

File: site-cms/wordpress/wp-content/themes/finansportalen/external/js/common/common-animations.js [hunk 2]
```diff
@@ -1,30 +1,30 @@
-angular.module('common')
```
Comment (ssinica) [inline site-cms/wordpress/wp-content/themes/finansportalen/external/js/common/common-animations.js]: Reviewer, pls ignore this. The file will not be touched during commit.

**Comment by leonardinius** [comment]
>Your comments are welcome. Pls, ignore commits history. Everything will be squash-ed before merging to master.

I see two problems with that
1. Code commit messages are (should be) in scope of review. E.g. does commit (CVS) properly describes the change introduced, does it include issue number et cetera... 
2. The code reviewed / code state is not particularly the one being merged into the master. 

If the final state to be merged is single, squashed commit - why should I bother at all to review / see the commit mess with all different sorts of WIP commit messages? 

Why not squash them already via `rebase -i` altogether and force-push here? In that case we could both judge/review commit message and see the final state as it will be merged.

**Comment by leonardinius** [comment]
> Reviewer, pls, ignore these change. I will not commit it.

IMO it's ok for WIP kind of reviews. If we are talking of something in the lines of "let's review it an merge in", then I have a reasonable doubt about that will be actually merged in.

* After reading this comment it's pretty clear it's not the code I see here.
* So, it must be different, modified  version of it?
* Should I re-do my code review afterwards?
* If no - then it's more WIP kind of code review procedure.
* If yes - why so? Isn't my time valuable too? It was pretty clear upfront the code change is needed / will be needed  before **actual review / merge **

**Comment by leonardinius** [comment]
Please don't read my comments as some form of tongue in the cheek / smart-butt $0.02 centness ;) I've just seen the skype chat regarding renewing semi-formal code review procedures and wanted to highlight potential upfront problems before it might take an unexpected direction: scope and procedure.

It's up to you to decide what kind of review is this going to be: 
a. very lightweight, WIP kind of (e.g. this is +- the code I'm working on at the moment). Please take a look, comment on it, maybe I need to take a different approach etc *OR* 
b. semi-formal (here is the BETA/final version of that I've worked on last few days/weeks. Please review & approve. We will merge it in ..)

**Comment by ssinica** [comment]
@leonardinius thanks for your input! We will discuss code review process tomorrow on scrum. Probably we will come up with some rules and understanding how we should do it. This pull request is just a beginning..., so please don't judge too hard :)

**Pull request closed without merging**